adafruit_is31fl3731.py

```diff
@@ -193,15 +265,25 @@ def fill(self, color=None, blink=None, frame=None):
             for col in range(18):
                 self._register(frame, _BLINK_OFFSET + col, data)
 
-    def _pixel_addr(self, x, y):
+    def pixel_addr(self, x, y): #pylint: disable-msg=no-self-use
+        """Calulate the offset into the device array for x,y pixel
+        """
         return x + y * 16
 
+    #pylint: disable-msg=too-many-arguments
     def pixel(self, x, y, color=None, blink=None, frame=None):
+        """Set blink or brightness for an x,y pixel
+            :param x: horizontal pixel position
+            :param y: vertical pixel position
+            :param color: brightness value 0->255
+            :param blink: True to blink
+            :param frame: the frame to set the pixel
+        """
         if not 0 <= x <= self.width:
             return
```

Comment:
Change this to `return None` to get rid of the "inconsistent return statement" warning

Reply:
ok